Lib9c.GraphQL/Types/PlanetNameType.cs

```diff
@@ -0,0 +1,10 @@
+using HotChocolate.Types;
+
+namespace Lib9c.GraphQL.Types;
+
+public class PlanetNameType : StringType
```

Comment:
If you want to limit the planet, how about implement it an enum?

Reply:
I sympathize with this sentiment. However, I have a few concerns
1. the convention of enums is to capitalize them. This is because the names of planets registered in the [planet registry](https://planets.nine-chronicles.com/planets/) are lowercase. For example, it would be awkward to treat "odin" as "PlanetName.ODIN" or "PlanetName.Odin".
2. Each planet is divided into main and internal networks, and the planet name is different for each network. Planet Odin's name on the internal network is "odin (internal)". So it was awkward to handle it as "PlanetName.ODIN_INTERNAL" or "PlanetName.OdinInternal".([planet-internal registry](https://planets-internal.nine-chronicles.com/planets/))
3. I didn't feel the need to restrict the planet name to an enum because of these concerns.

However, I think it's an advantage to treat it as an enum to reduce mistakes from the user's point of view, so I'll write a sufficient description in the schema and treat it as an enum.

> 저도 공감합니다. 그런데 고민이 되는 부분이 있습니다.
> 1. enum의 컨벤션 상 대문자로 처리하는 것이 걸립니다. 그 이유는 [플레닛 레지스트리](https://planets.nine-chronicles.com/planets/)에 등록된 플레닛의 이름은 소문자이기 때문입니다. 예를 들어서 "odin"을 "PlanetName.ODIN" 또는 "PlanetName.Odin"으로 처리하는 게 어색합니다.
> 2. 각 플래닛은 main과 internal 네트워크로 나뉘어져 있는데, 네트워크에 따라서 플레닛 이름이 다릅니다. 오딘 플레닛의 internal 네트워크에서의 이름은 "odin (internal)"인데, 이걸 "PlanetName.ODIN_INTERNAL" 또는 "PlanetName.OdinInternal"로 처리하는 게 어색합니다.([planet-internal registry](https://planets-internal.nine-chronicles.com/planets/))
> 3. 이런 고민이 있어서 플레닛 이름을 굳이 enum으로 제한할 필요를 느끼지 못했습니다.
> 
> 그래도 enum으로 처리하면 사용자 입장에서 실수를 줄일 수 있다는 점은 장점이라고 생각하기 때문에 충분한 설명을 스키마에 작성하고 enum으로 처리해두기로 하겠습니다.

Reply:
Applied!